### PR TITLE
Misc/extend php cs fixer configuration

### PR DESCRIPTION
**Pull request by teiling88**
- satisfy php doc comments
- sort use statements
- replace double quotes with single quotes

### CODE REVIEW

File: src/Application/DependencyInjection.php [hunk 1]
```diff
@@ -11,6 +11,7 @@ class DependencyInjection
 {
     const DEFAULT_PROCESS_TIMEOUT = 180;
     /**
+     * @param mixed $rootDir
```
Comment (mitelg) [inline src/Application/DependencyInjection.php]: mixed is not very nice

File: src/Application/ExtensionManager.php [hunk 2]
```diff
@@ -45,6 +45,7 @@ public function __construct(ClassLoader $autoLoader)
     /**
      * Read all available plugins
      *
+     * @param mixed $extensionDirs
```
Comment (mitelg) [inline src/Application/ExtensionManager.php]: mixed

File: src/Cache/CacheInterface.php [hunk 3]
```diff
@@ -20,19 +20,16 @@ public function write($key, $data, $valid);
...
     public function read($key);
 
-    /**
-     * @return void
-     */
+    
     public function delete($key);
 
     /**
+     * @param mixed $key
```
Comment (mitelg) [inline src/Cache/CacheInterface.php]: mixed

File: src/Extensions/Shopware/DataGenerator/Command/CreateDataCommand.php [hunk 4]
```diff
@@ -156,6 +156,7 @@ public function interact(InputInterface $input, OutputInterface $output)
      * @param InputInterface $input
      * @param string optionName
      * @param string|null $optionHumanName
+     * @param mixed $optionName
```
Comment (mitelg) [inline src/Extensions/Shopware/DataGenerator/Command/CreateDataCommand.php]: mixed

File: src/Extensions/Shopware/DataGenerator/Command/CreateDataCommand.php [hunk 5]
```diff
@@ -176,16 +177,17 @@ private function askConfigOptions(InputInterface $input, $optionName, $optionHum
 
     /**
      * @param  string
-     * @return int
+     * @param mixed $input
```
Comment (mitelg) [inline src/Extensions/Shopware/DataGenerator/Command/CreateDataCommand.php]: mixed

File: src/Extensions/Shopware/DataGenerator/Writer/BufferedFileWriter.php [hunk 6]
```diff
@@ -37,6 +37,7 @@ class BufferedFileWriter implements WriterInterface
 
     /**
      * @param string $file
+     * @param mixed $maxBufferSize
```
Comment (mitelg) [inline src/Extensions/Shopware/DataGenerator/Writer/BufferedFileWriter.php]: mixed

File: src/Extensions/Shopware/Install/Services/Install/Vcs.php [hunk 7]
```diff
@@ -108,8 +108,8 @@ private function getDestinationPath($installDir, $destination)
     /**
      * Enforce a configured core repository
      *
-     * @return mixed
      * @throws \RuntimeException
+     * @return mixed
```
Comment (mitelg) [inline src/Extensions/Shopware/Install/Services/Install/Vcs.php]: mixed

File: src/Extensions/Shopware/Plugin/Services/Zip.php [hunk 8]
```diff
@@ -62,6 +62,7 @@ public function zip(Plugin $plugin, $path, $zipTo, $branch, $useHttp = false)
 
     /**
      * @param string $directory
+     * @param mixed $outputFile
```
Comment (mitelg) [inline src/Extensions/Shopware/Plugin/Services/Zip.php]: mixed

File: src/Extensions/Shopware/PluginCreator/Command/CreatePluginCommand.php [hunk 9]
```diff
@@ -229,8 +229,8 @@ public function upperToArray($input)
      * Make sure the namespace is one of core, backend, frontend
      *
      * @param $input
-     * @return mixed
      * @throws \InvalidArgumentException
+     * @return mixed
```
Comment (mitelg) [inline src/Extensions/Shopware/PluginCreator/Command/CreatePluginCommand.php]: mixed

File: src/Extensions/Shopware/PluginCreator/Command/CreatePluginCommand.php [hunk 10]
```diff
@@ -245,8 +245,8 @@ public function validateNamespace($input)
      * Check the entered model (check might be somewhat more sufisticated)
      *
      * @param $input
-     * @return mixed
      * @throws \InvalidArgumentException
+     * @return mixed
```
Comment (mitelg) [inline src/Extensions/Shopware/PluginCreator/Command/CreatePluginCommand.php]: mixed

**Comment by teiling88** [comment]
i fixed it but what about all the other mixed types in other classes...

File: src/Extensions/Shopware/DataGenerator/Command/CreateDataCommand.php [hunk 11]
```diff
@@ -176,16 +177,17 @@ private function askConfigOptions(InputInterface $input, $optionName, $optionHum
 
     /**
      * @param  string
-     * @return int
+     * @param $input
```
Comment (mitelg) [inline src/Extensions/Shopware/DataGenerator/Command/CreateDataCommand.php]: now you have 2 params here

**Comment by mitelg** [comment]
they should be fixed by someone someday :smile:

**Comment by teiling88** [comment]
next year maybe? :-)

File: src/Extensions/Shopware/DataGenerator/Command/CreateDataCommand.php (hunk 11 above)
Comment (teiling88) [inline src/Extensions/Shopware/DataGenerator/Command/CreateDataCommand.php]: surprise ;-)

**Comment by mitelg** [comment]
i am awaiting your PR @teiling88

**Comment by teiling88** [comment]
image

**Pull request merged**